cryptoservice/crypto_service.go

```diff
@@ -58,16 +64,29 @@ func (ccs *CryptoService) Create(role string, algorithm data.KeyAlgorithm) (*dat
 	if err != nil {
 		return nil, fmt.Errorf("failed to add key to filestore: %v", err)
 	}
-	return data.PublicKeyFromPrivate(*privKey), nil
+	return data.PublicKeyFromPrivate(privKey), nil
+}
+
+// GetKey returns a key by ID
```

Comment:
Got it. Lets keep it this way for now. Maybe we'll see some other abstraction that makes sense in the future.